crypto/encrypt/encrypt_test.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright © 2021 Sniptt <support@sniptt.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package encrypt
+
+import (
+	"bytes"
+	"crypto/aes"
+	"crypto/cipher"
+	"encoding/hex"
+	"regexp"
+	"testing"
+)
+
+func TestBytes(t *testing.T) {
+	secret := "nuclear_launch_codes"
+	want := regexp.MustCompile(secret)
+	bytesToEncrypt := []byte(secret)
+	key, _ := hex.DecodeString("6368616e676520746869732070617373776f726420746f206120736563726574")
+
+	// Encrypt secret.
+	ciphertext, secretKey, nonce, err := Bytes(key, bytesToEncrypt)
+
+	// Remove nonce from start of ciphertext.
+	ciphertext = ciphertext[len(nonce):]
+
+	if bytes.Compare(key, secretKey) != 0 || err != nil {
+		t.Fatalf("Expected %v, got %v", key, secretKey)
+	}
+
+	block, err := aes.NewCipher(key)
+	if err != nil {
+		t.Fatalf(err.Error())
+	}
+
+	aesgcm, err := cipher.NewGCM(block)
```

Comment:
Here it's `aesgcm` but in the original file has been updated to `aesGCM` I believe.